docs/python/python-tutorial.md

```diff
@@ -30,6 +30,8 @@ To successfully complete this tutorial, you must do the following:
 
 1. On Windows, make sure the location of your Python interpreter is included in your PATH environment variable. You can check this by running `path` at the command prompt. If the Python interpreter's folder isn't included, open Windows Settings, search for "environment", select **Edit environment variables for your account**, then edit the **Path** variable to include that folder.
 
+1. On macOS, make sure the location of your VS Code installation is included in your PATH environment variable.  See [the setup instructions](/docs/setup/mac#_preferences-menu) for more information.
```

Comment:
I think you want to link to "Launching from the Command Line" which would be /docs/setup/mac.md#launching-from-the-command-line. Note the use of `mac.md#launching...` as this will let people navigate on GitHub and also be correctly linked on the code.visualstudio.com site.

Reply:
You are absolutely right.  Thank you.  I will update.